src/omniperf_profile/profiler_base.py

```diff
@@ -365,35 +383,36 @@ def run_profiling(self, version: str, prog: str):
                 )
                 # log output from profile filtering
                 if not success:
-                    error(output)
+                    console_error(output)
                 else:
-                    logging.debug(output)
-            logging.info("\nCurrent input file: %s" % fname)
+                    console_debug(output)
+            console_log("profile", "Current input file: %s" % fname)
```

Comment:
Everywhere else we've been using the "profiling" keyword, as opposed to "profile" in logging output. Consider changing for consistency
```suggestion
            console_log("profiling", "Current input file: %s" % fname)
```